package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Priceline Design System",
   "main": "dist/index.js",
   "scripts": {
-    "prepare": "npm run icons && babel src -d dist --ignore __tests__",
+    "prepublish": "npm run icons && babel src -d dist --ignore __tests__",
```

Comment:
For my education, what would be the reason to use “prepublish” instead of “prepare”. I was under the assumption that later NPM version favors “prepare” over “prepublish”. Would like to use any special use case for “prepublish”.

Reply:
Yeah, we talked about this in the common components meeting, but because some people here are still using npm 3, using `prepublish` is a safer bet that the package will build as expected. If we get most people up to npm 5, then we can switch back to `prepare`